internal/component/kafka/consumer.go

```diff
@@ -101,6 +184,40 @@ func (k *Kafka) RemoveTopicHandler(topic string) {
 	k.subscribeLock.Unlock()
 }
 
+// AddBulkSubscribeConfig adds bulk subscribe config
+func (k *Kafka) AddBulkSubscribeConfig(maxBulkCount int, maxBulkAwaitDurationMilliSeconds int, maxBulkSizeBytes int) {
+	k.bulkSubscribeConfig.MaxBulkCount = maxBulkCount
+	k.bulkSubscribeConfig.MaxBulkAwaitDurationMilliSeconds = maxBulkAwaitDurationMilliSeconds
+	k.bulkSubscribeConfig.MaxBulkSizeBytes = maxBulkSizeBytes
```

Comment:
These two options are not used as of now.

Reply:
MaxBulkAwaitDurationMilliSeconds is being used now - removed MaxBulkSizeBytes as it is not used as of now.